### ISSUE TITLE
Major audio playback issues on alioth

### ISSUE THREAD

**Issue by Sentosa2012** [PixelExperience/android-issues#2183]
## Build date

PixelExperience_alioth-12.1-20220331-0705-OFFICIAL

## Expected Behavior

Audio should playback smoothly

## Current Behavior

Major skipping, UI freezing, and sometimes unresponsive volume rocker

## Possible Solution

No clue, not really familiar with the inner-workings of Android. I do have logs and a video example but it seems the validation bot doesn't like me including them here. 

## Steps to Reproduce

1. Install any currently available build of PE
2. Start playing a video or audio
3. Continue using the device normally (change volume, open other apps, lock device, etc.)

/device alioth
/version twelve

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by Sentosa2012** [comment]
[logcat.txt](https://github.com/PixelExperience/android-issues/files/8448020/logcat.txt)

https://user-images.githubusercontent.com/47618761/162350825-a0ad986b-34a9-46cd-a842-31294bd579a7.mp4

**Comment by lzgmc** [comment]
That's odd. I just checked and Spotify works fine on my device, as well as YouTube, YT Music, and other media consumption services, without weird glitches or stutters. Did you do a clean flash from the latest build of MIUI 12.5?

**Comment by Sentosa2012** [comment]
No, I came straight from PE based on Android 11. Do you have a link to download the stock ROM? The websites I've found seem a bit sketchy to me

**Comment by lzgmc** [comment]
I believe that you can edit the bat file to remove the anti rollback checks and downgrade. Look for guides on the internet for instructions. Alternatively, you can try flashing a xiaomi.eu build of MIUI 12.5, as those have no anti-rollback checks.

**Comment by Sentosa2012** [comment]
Sure I'll give that a go and see what happens. Knowing me I'll probably screw something up but that's why is a testing phone haha

**Comment by Sentosa2012** [comment]
> Also seeing this issue on apollo/apolloPro (Mi 10T Pro) - did updating the stock firmware (presumably using the pre-built zip with flash scripts) help @Sentosa2012 ?

So far it looks like it did fix the issues I was having. I only got round to reflashing my device a couple days ago so I'm not comfortable saying there is no more problems but so far so good!

The steps I took are as follows:
1. Install the latest version of MiFlash
2. Download the latest fastboot version of MiUI 12.5
3. Remove the antirollback section of the flash_all.bat script
4. Reboot into bootloader/fastboot
5. Flash the MiUI fastboot ROM via MiFlash
6. Reflash PixelExperience according to your device's instructions

Just be aware some steps may differ on your device
